### PR TITLE
Add side buttons to change PDF pages [DISCL-315]

### PR DESCRIPTION
**Pull request by rdumusc**
Also show the previous/next buttons for webbrowsers

### CODE REVIEW

**Comment by jafyvilla** [comment]
I'm not too familiar with qml, but looks good to me. +1

File: tide/core/PixelStreamContent.h [hunk 1]
```diff
@@ -47,12 +47,16 @@
...
 {
+    Q_OBJECT
+    Q_PROPERTY( int page READ getPage CONSTANT )
+    Q_PROPERTY( int pageCount READ getPageCount CONSTANT )
+
 public:
     /**
      * Constructor.
      * @param uri The unique stream identifier.
      */
```
Comment (tribal-tec) [inline tide/core/PixelStreamContent.h]: missing doxygen for new parameter

File: tide/core/PixelStreamContent.h [hunk 2]
```diff
@@ -66,6 +70,12 @@ class PixelStreamContent : public Content
     /** @return true if the streamer can handle aspect ratio changes. */
     bool hasFixedAspectRatio() const override;
 
+    /** Get the current page number. */
+    int getPage() const;
+
+    /** Get the total number of pages. */
```
Comment (tribal-tec) [inline tide/core/PixelStreamContent.h]: page and pageCount for pixelsteams??

Comment (rdumusc) [inline tide/core/PixelStreamContent.h]: It is only used by the webbrowser and I think it makes sense in this case. I didn't find a more generic name for this. Maybe subclassing PixelStreamContent with a new WebbrowserContent and adding the two functions there would be better?

**Pull request merged**